module/sheets/base-actor-sheet.js

```diff
@@ -90,7 +90,7 @@ export default class DGActorSheet extends DGSheetMixin(ActorSheetV2) {
 
     // Set title for Physical Attributes.
     context.physicalAttributesTitle = game.i18n.localize(
-      "DG.Sheet.BlockHeaders.Attributes",
+      "DG.Sheet.BlockHeaders.Statistics",
```

Comment:
Header for statistics, it currently says "Attributes", which are derived stats, that the section is for Statistics